src/pubsub/mod_pubsub_db_rdbms.erl

```diff
@@ -424,3 +449,29 @@ int2sub(1) -> pending;
 int2sub(2) -> unconfigured;
 int2sub(3) -> subscribed.
 
+item_to_record({NodeIdx, ItemId, CreatedLUser, CreatedLServer, CreatedAt,
+                ModifiedLUser, ModifiedLServer, ModifiedLResource, ModifiedAt,
+                PublisherIn, PayloadDB}) ->
+    PayloadXML = mongoose_rdbms:unescape_binary(CreatedLServer, PayloadDB),
+    {ok, #xmlel{children = Payload}} = exml:parse(PayloadXML),
+    ItemAndNodeId = {ItemId, mongoose_rdbms:result_to_integer(NodeIdx)},
+    Creation = {usec:to_now(mongoose_rdbms:result_to_integer(CreatedAt)),
+                {CreatedLUser, CreatedLServer, <<>>}},
```

Comment:
Are we certain that creator JID is always bare in PubSub logic?

Reply:
According to our tests and the code it seams so.